mojo/core/broker_castanets.h

```diff
@@ -77,9 +81,9 @@ class BrokerCastanets : public Channel::Delegate, public base::SyncDelegate {
   bool IsHost() const { return host_; }
 
   BrokerCastanets(base::ProcessHandle client_process,
-             ConnectionParams connection_params,
-             const ProcessErrorCallback& process_error_callback,
-             CastanetsFenceManager* fence_manager);
+                  ConnectionParams connection_params,
+                  const ProcessErrorCallback& process_error_callback,
+                  CastanetsFenceManager* fence_manager);
```

Comment:
I've already applied the same code cleanup. Don't put code for cleanup in this pr. One commit = one issue.

Reply:
I checked your commit below, and this change made by 'git cl format' was not applied in the commit below.
This style change was made automatically by script(although I didn't intent to do that), so I guess it's ok to go.

d5e7ff134cf743a39c7ba11618afb20c373fc104
Author: Hyowon Kim <hw1008.kim@samsung.com>
Date:   Wed Sep 11 10:07:14 2019 +0900

    Code cleanup in BrokerCastanets.